other/web-js/client/src/components/Theme.js

```diff
@@ -37,7 +45,6 @@ switch (API_TYPE) {
     };
     break;
   }
-  // :state-end:
   default: {
     throw new Error(
       `Invalid REACT_APP_API_TYPE: "${API_TYPE}". Specify "graphql", "mql", or "local" instead.`
```

Comment:
> Specify "graphql", "mql", or "local" instead.

these API types probably should come from app metadata rather than be hardcoded here